sys/pylon/gstpylonsrc.c

```diff
@@ -198,6 +230,12 @@ ascii_strdown(gchar* * str, gssize len)
   *str = temp;
 }
 
+static inline _Bool
+fnequal(double val, double def)
```

Comment:
I'd make a more verbose function name or put a comment. It seems to me this function checks if `val` is within 1% of `def`.

Reply:
Now I realize it's the inverse of what I said above.

Curious how this came about? Including the choice of 1%. Can any of the features be set closer than 1% of the default value?

Reply:
I was trying to avoid exact floating point comparsion as it is error prone. Probably it is not the best solution.
I can revert it back to original if you wish. Any way if you'll pull my further changes with property states, there is no need in this comparsion anymore. As it is trying to check if the value have been set explicitly or left default

Reply:
Ok